src/main/java/appeng/util/item/FuzzyItemVariantList.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * This file is part of Applied Energistics 2.
+ * Copyright (c) 2013 - 2020, AlgorithmX2, All rights reserved.
```

Comment:
We should move the new or nearly completey files to `Copyright (c) 2021, TeamAppliedEnergistics, All rights reserved.`. Or whatever year will apply.